SPECS/sriov-network-device-plugin/sriov-network-device-plugin.spec

```diff
@@ -0,0 +1,37 @@
+Summary:        Plugin for discovering and advertising networking resources
+Name:           sriov-network-device-plugin
+Version:        3.4.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin
+Source0:        https://github.com/k8snetworkplumbingwg/%{name}/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+BuildRequires:  golang
+
+%description
+sriov-network-device-plugin is Kubernetes device plugin for discovering and advertising networking
+resources in the form of SR-IOV virtual functions and PCI physical functions
+
+%prep
+%autosetup -p1
+
+%build
+go build -mod vendor -o ./build/sriovdp ./cmd/sriovdp/
+
+%install
+install -D -m0755 build/sriovdp %{buildroot}%{_bindir}/sriovdp
+install -D -m0755 images/entrypoint.sh %{buildroot}%{_bindir}/entrypoint.sh
+install -D -m0755 images/ddptool-1.0.1.12.tar.gz %{buildroot}%{_bindir}/ddptool-1.0.1.12.tar.gz
```

Comment:
I'd like to know a bit more about this tool. Why are we putting the source for a different tool into this tool's final RPM? Why not put it in a separate package where we can ship it already built and update it independently for security fixes, bug fixes, packaging fixes, etc.?

Reply:
Also, same comment as for the entrypoint script- if we do actually need to ship the tarball like this, it would make more sense under `/usr/share/%{name}`.

Reply:
Yes, I am able to put the tarbell under /usr/share/%{name}/ddptool-1.0.1.12.tar.gz - would that work to resolve this

Reply:
That's really a solution of last resort for us. We'd really prefer it not to be there at all and ship ddptool as its own package, built with all of our macros and security hardening and CVE tracking.

Reply:
@romoh @liulanze @yapancha any ideas on where best to place these files: the `entrypoint.sh` and the 'ddptool' tarbell?

Reply:
The right fix here IMO is in the sriov-dp repo. ddptool tool shouldn't be consumed that way, it should be pulled as a dependency and built. Maybe something we can discuss with upstream sriov-dp folks. Till then, I think we should replicate the upstream project. Putting things under `/usr/share` sounds like a good mitigation.